drivers/pm_fallback/set_lowest.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_periph_pm
+ * @{
+ *
+ * @file
+ * @brief       Platform-independent power management fallback code
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include "irq.h"
+#include "periph/pm.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+void pm_set_lowest(void) {}
```

Comment:
I don't like this file much... ;)

Reply:
Me neither, but that's the fallback and it was before as well. Any suggestions?

Reply:
Not with using submodules and without using ifdefs. Let's keep it for now.

Reply:
pm_set_lowest is only used in `core/kernel_init.c` and `sys/pm_layered/pm.c`, maybe we can omit the file and only call pm_set_lowest in `kernel_init.c` if the CPU/board defines something like `#define HAS_PM_SET_LOWEST` or something?